test/check-machines-create

```diff
@@ -378,20 +378,21 @@ class TestMachinesCreate(VirtualMachinesCase):
         m.execute(f"echo 'vnc_password= \"{vnc_passwd}\"' >> /etc/libvirt/qemu.conf")
 
         runner.createAndVerifyQemuConfParsedTest(
-                TestMachinesCreate.VmDialog(self, sourceType='file',
-                                            storage_pool="NoStorage",
-                                            location=config.NOVELL_MOCKUP_ISO_PATH),
-                vnc_listen, "127.0.0.1", vnc_passwd, None)
+            TestMachinesCreate.VmDialog(self, sourceType='file',
+                                        storage_pool="NoStorage",
+                                        location=config.NOVELL_MOCKUP_ISO_PATH),
+            vnc_listen, "127.0.0.1", vnc_passwd, None)
```

Comment:
Unrelated whitespace change. Would be better as separate commit. (Same a bit below)

Reply:
Sorry about that, I made my editor lint/fix things which usually goes fine except when it doesn't :)